app/components/SuccessMessage.tsx

```diff
@@ -0,0 +1,47 @@
+// LIBRARY IMPORTS
+import { Box, Button, Icon, IconButton, Text, VStack } from "@chakra-ui/react";
+import { FaRegPaperPlane, FaXmark } from "react-icons/fa6";
+
+// TYPE DEFINITIONS
+type SuccessMessageProps = {
+  handleCloseSuccess: () => void;
+};
+
+// SUCCESS MESSAGE COMPONENT DEFINITION
+export default function SuccessMessage({
+  handleCloseSuccess,
+}: SuccessMessageProps) {
+  return (
+    <Box position="relative" py={16}>
```

Comment:
The only thing that I noticed with success form is that it doesn't stretch vertically all the way like the quote form. That's what it looks like for 1728x1117 screen size. Also, the spacing between the form and content seems to be quite large.
![Screen Shot 2024-02-27 at 8 13 33 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/7756459e-bda3-4c62-9f49-b0b52a4bdd9b)